OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -102,19 +103,24 @@ public virtual void Tick(Actor self)
 			// NOTE: FaceTarget is called in AttackTurreted.CanAttack if the turret has a target.
 			if (attack != null)
 			{
-				if (!attack.IsAttacking)
+				if (!attack.IsAttacking || attack.IsTraitDisabled)
 				{
-					if (realignTick < info.RealignDelay)
-						realignTick++;
-					else if (info.RealignDelay > -1)
-						DesiredFacing = null;
-
-					MoveTurret();
+					attack = null;
 				}
```

Comment:
Convention was to have no curly brackets around if statements if they are a single line and don't contain an if statement inside and else outside.